### PR TITLE
chore: seo updates

### LINKED ISSUE #232: optimize for SEO
based on the audit from Victorious, we should optimize for keywords

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

We're optimizing based on keywords from Victorious and also updating the paths to use /learn and /docs.

## Related Tickets & Documents

closes #232

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

**Review by jpmcb — APPROVED** [review]

File: docs/becoming-a-maintainer/your-team.md [hunk 1]
```diff
@@ -142,11 +142,11 @@ Here are some things to consider:
 - **Goals:** Clearly define the project's goals and how the team members can contribute. This will help them to understand how they can contribute to the project's success.
 - **Timeline:** Set clear expectations for the timeline. This will help them understand what to do and the deadlines.
 
-One way to onboard your new team members is to have clear guidelines and include them in your documentation. If you need an idea for creating one, you can look at the [OpenSauced Community Maintainers Guidelines](https://docs.opensauced.pizza/contributing/opensauced-maintainers-guide/community-maintainers-guide/) or read [this blog post](https://dev.to/opensauced/the-missing-piece-why-your-project-needs-a-maintainer-onboarding-process-np0).
+One way to onboard your new team members is to have clear guidelines and include them in your documentation. If you need an idea for creating one, you can look at the [OpenSauced Community Maintainers Guidelines](https://opensauced.pizza/docs/contributing/opensauced-maintainers-guide/community-maintainers-guide/) or read [this blog post](https://dev.to/opensauced/the-missing-piece-why-your-project-needs-a-maintainer-onboarding-process-np0).
```
Comment (nickytonline) [inline docs/becoming-a-maintainer/your-team.md:145]: Same comment about the dev.to link. Consider replacing it with a community resource blog post link if one exists.

**Review by nickytonline — APPROVED** [review]
Looks great! A few suggestions/comments.

File: docs/becoming-a-maintainer/your-team.md (hunk 1 above)
Comment (BekahHW) [reply docs/becoming-a-maintainer/your-team.md:145]: This isn't in there yet.

File: docs/becoming-a-maintainer/building-community.md [hunk 2]
```diff
@@ -91,11 +91,11 @@ Some of your contributors may have skills that can be useful to promote your pro
...
+Issue management involves identifying, analyzing, and prioritizing bug issues and feature requests during development. You can create a triage team and encourage your community to help you triage and prioritize the issues in your repositories. You want to equip them with a clear guide on managing issues like what we have at OpenSauced in our [Triage Guide](https://opensauced.pizza/docs/contributing/triage-guide/). You can also read [this article](https://dev.to/opensauced/collaborate-conquer-grow-mastering-the-art-of-issue-management-for-open-source-projects-49gi) to learn more about issue management.
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/becoming-a-maintainer/building-community.md:94]: Not in there yet.

File: docs/becoming-a-maintainer/building-community.md [hunk 3]
```diff
@@ -7,29 +7,29 @@ keywords: ["building community", "building open source community", "nurturing co
...
 
-As more people engage and contribute to your projects, it becomes crucial for you to provide them with the necessary support and motivation to establish a welcoming and positive community environment. Doing so can build stronger relationships and promote a sense of belonging among your community members, keeping them returning and remaining dedicated.
+As more people engage and contribute to your projects, it becomes crucial for you to provide them with the necessary support and motivation to establish a welcoming and positive open source community environment. Doing so can build stronger relationships and promote a sense of belonging among your community members, keeping them returning and remaining dedicated.
 
-In this chapter, you will learn how to build and nurture a welcoming and supportive community in open source.
+In this chapter, you will learn how to build and nurture a welcoming and supportive open source community in open source.
```
[the remark this answers is not in the document]
Comment (BekahHW) [reply docs/becoming-a-maintainer/building-community.md]: Updated!

**Pull request merged**